source/sass/mozfest.scss

```diff
@@ -1,15 +1,69 @@
 body.mozfest {
   &#view-mozfest-home {
     #hero {
-      --moz_banner_height: 470px;
+      --moz_banner_height: 464px;
```

Comment:
We are utilizing many SCSS variables from the main site and Bootstrap here already. For consistency purposes I wonder if we should just stick with SCSS variables instead of CSS's custom properties?

Reply:
That's fine, let's go with scss.